update base-primitives.html

nshen opened the pull request:
I added  Cylinder and Triangle primitives to the example.  The triangle looks a little weird, but I think it works.

gordonnl commented:
Thanks for this.

The `Triangle` class is actually meant to be used in this sense.

The purpose of triangle is to create a quad (for full screen effects or sprites) using only a single triangle.

![glscreenspacetriangle1](https://user-images.githubusercontent.com/3521882/75447068-e6179f00-5968-11ea-9cf8-7a3cbbd5117a.png)

This has a few benefits over the old method of using two triangles.

This is also why the position is a vec2 and not a vec3.

I understand the confusion, and I honestly couldn't think of a better name, but am very much open to a name change!

nshen commented:
Thanks for reply .
you are right, I just updated the pull request, I removed the triangle mesh, but leave the Cylinder there.
I'm thinking Is there a way to make triangle class both as a mesh and use for full screen effects ?

gordonnl commented:
Thanks mate.

I do currently use it for sprites as well (basically wherever I'd usually use a Plane as a quad). If you need a 3D projection, in your vertex shader you just need to convert the vec2 to a vec3:

```
vec3 pos = vec3(position, 0);
```
